### PR TITLE
New projects feature - work in progress flag

### PR DESCRIPTION
**Pull request by BrightKrystian**

### CODE REVIEW

File: src/components/shared/StatusPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      <CustomSectionInner>
+        <CustomTextRegular centered>
+          {children}
+        </CustomTextRegular>
+      </CustomSectionInner>
+    </CustomSection>
+  </>
+)
+
+export default StatusPanel
```
Comment (miensol) [inline src/components/shared/StatusPanel.tsx]: I think we can use regular exports, unless there's a good reason for using default exports?

File: src/components/shared/index.tsx [hunk 2]
```diff
@@ -78,3 +78,5 @@ export const CustomSectionInner = Styled.CustomSectionInner
 export const CustomPageDescription = Styled.CustomPageDescription
 export const BlackButton = Styled.BlackButton
 export const FlexWrapper = Styled.FlexWrapper
+
+export * as StatusPanel from './StatusPanel'
```
Comment (miensol) [inline src/components/shared/index.tsx]: Is that used anywhere?

**Review by miensol — APPROVED** [review]

File: src/components/shared/StatusPanel.tsx (hunk 1 above)
Comment (BrightKrystian) [reply src/components/shared/StatusPanel.tsx]: Changed to named export

**Review by miensol — APPROVED** [review]

File: src/components/shared/index.tsx (hunk 2 above)
Comment (BrightKrystian) [reply src/components/shared/index.tsx]: It is now, the imports are changed to used barrel file

**Pull request merged**